app/routes/news._index.mdx

```diff
@@ -26,6 +26,35 @@ import { Anchor } from '~/components/Anchor'
 
 ## 2024
 
+    <CollectionItem
+      className="maxw-none"
+      variantComponent={
+        <CollectionCalendarDate datetime={'May 20, 2024'} />
+      }
+    >
+      <CollectionHeading headingLevel="h3">
+        <Anchor>New Einstein Probe Notices, Circulars Astro-Flavored Markdown, and Notices Schema v4.0.0</Anchor>
+      </CollectionHeading>
+      <CollectionDescription>
+
+        - **Einstein Probe** new GCN notice type is now available in JSON format over Kafka (https://gcn.nasa.gov/quickstart), but are not yet available via email.  They are not available via GCN Classic.
+          - Einstein Probe (EP) Summary: EP is a mission of the Chinese Academy of Sciences (CAS), in collaboration with the European Space Agency (ESA) and the Max Planck Institute for extraterrestrial Physics (MPE), Germany, dedicated to time-domain high-energy astrophysics. Its primary goals are to discover high-energy transients and monitor variable objects.
```

Comment:
```suggestion
          - Einstein Probe (EP) Summary: EP is a mission of the Chinese Academy of Sciences (CAS), in collaboration with the European Space Agency (ESA) and the Max Planck Institute for Extraterrestrial Physics (MPE), Germany, dedicated to time-domain high-energy astrophysics. Its primary goals are to discover high-energy transients and monitor variable objects.
```

Reply:
Add hyperlinks for all of these other agencies.